src/api/java/com/minecolonies/api/crafting/RecipeStorage.java

```diff
@@ -238,6 +243,11 @@ public boolean fullfillRecipe(final List<IItemHandler> handlers)
 
         for (final ItemStorage stack : getCleanedInput())
         {
+            if (stack.getItem() == ModItems.buildTool)
```

Comment:
same as previous

Reply:
This is so that it doesn't get "discounted" because the tool is not in the inventory

Reply:
Ah wait, we call cleaned input, you're right

Reply:
I'll adjust it